### PR TITLE
[4.5.0] [Docker-swarm] Software Update: ELK v8.7.0

### PR DESCRIPTION
**Pull request by karun-singh**
- Moved to bitnami elk images
- Updated elk stack to v8.7.0

### CODE REVIEW

File: Docker-Swarm-deployment/single-node/elk/README.md [hunk 1]
```diff
@@ -16,34 +16,42 @@ Docker swarm stack for Elasticsearch-logstash-Kibana (ELK) Deployment.
...
 ├── keystores
-│   ├── elasticsearch.keystore
 │   ├── kibana.keystore
 │   └── logstash.keystore
 ├── passwords
 │   ├── elasticsearch-cat-password
 │   ├── elasticsearch-fs-password
 │   ├── elasticsearch-rs-password
 │   ├── elasticsearch-su-password
+│   ├── es-password.env
```
Comment (abhi4578) [inline Docker-Swarm-deployment/single-node/elk/README.md:31]: what/how to define this env file? can u put an example file in example-secrets/secrets/passwords folder?

File: Docker-Swarm-deployment/single-node/elk/database-stack.yaml [hunk 2]
```diff
@@ -19,6 +19,14 @@ services:
...
+      - secrets/passwords/es-password.env
+      - secrets/passwords/snapshot-credentials.env
+    environment:
+      - ELASTICSEARCH_ENABLE_SECURITY=true
+      - ELASTICSEARCH_TLS_VERIFICATION_MODE=certificate
+      - ELASTICSEARCH_KEYSTORE_LOCATION=/opt/bitnami/elasticsearch/config/certs/elastic-certificates.p12
+      - ELASTICSEARCH_TRUSTSTORE_LOCATION=/opt/bitnami/elasticsearch/config/certs/elastic-certificates.p12
     configs:
       - source: elasticsearch-config
         target:  /usr/share/elasticsearch/config/elasticsearch.yml
```
Comment (abhi4578) [inline Docker-Swarm-deployment/single-node/elk/database-stack.yaml:32]: change volume name , and volume mount points change for elk?

**Review by abhi4578 — CHANGES_REQUESTED** [review]
I think u missed the first commit from this PR to cherry pick https://github.com/datakaveri/iudx-deployment/pull/448/commits/0fc300520937bfbd50bd81c3b10c906ff841b050. You can maybe do cherry picking again on base of 4.5.0 branch and open new pr.

File: Docker-Swarm-deployment/single-node/elk/README.md (hunk 1 above)
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/elk/README.md:31]: It is generated by the `create-secrets.sh` script, the user does not need to configure it manually.

File: Docker-Swarm-deployment/single-node/elk/database-stack.yaml (hunk 2 above)
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/elk/database-stack.yaml:32]: Mount point change was in the first commit, will be included in new PR. Will add the name change.

**Comment by karun-singh** [comment]
Missing commit from cherry-pick. Opened new PR #517

**Pull request closed without merging**